pkgs/applications/graphics/tesseract/default.nix

```diff
@@ -1,67 +1,82 @@
-                         then "1h48xfzabhn0ldbx5ib67cp9607pr0zpblsy8z6fs4knn0zznfnw"
-                         else null)
+# This argument is obsolete
+, enableLanguagesHash ? null
 }:
 
-let tessdata = stdenv.mkDerivation ({
-  name = "tessdata";
-  src = fetchFromGitHub {
-    owner = "tesseract-ocr";
-    repo = "tessdata";
-    rev = "3cf1e2df1fe1d1da29295c9ef0983796c7958b7d";
-    # when updating don't forget to update the default value fo enableLanguagesHash
-    sha256 = "1v4b63v5nzcxr2y3635r19l7lj5smjmc9vfk0wmxlryxncb4vpg7";
-  };
-  buildCommand = ''
-    cd $src;
-    for lang in ${if enableLanguages==null then "*.traineddata" else stdenv.lib.concatMapStringsSep " " (x: x+".traineddata") enableLanguages} ; do
-      install -Dt $out/share/tessdata $src/$lang ;
-    done;
-  '';
-  preferLocalBuild = true;
-  } // (stdenv.lib.optionalAttrs (enableLanguagesHash != null) {
-  # when a hash is given, we make this a fixed output derivation.
-  outputHashMode = "recursive";
-  outputHashAlgo = "sha256";
-  outputHash = enableLanguagesHash;
-  }));
-in
+let
+  languages = (import ./languages.nix { inherit stdenv fetchurl fetchFromGitHub; }).v3;
 
-stdenv.mkDerivation rec {
-  name = "tesseract-${version}";
-  version = "3.05.00";
+  tessdata = if enableLanguages == null then
+      languages.all
+    else
+      map (lang: languages.${lang}) enableLanguages;
 
-  src = fetchFromGitHub {
-    owner = "tesseract-ocr";
-    repo = "tesseract";
-    rev = version;
-    sha256 = "11wrpcfl118wxsv2c3w2scznwb48c4547qml42s2bpdz079g8y30";
-  };
+  tesseractWithoutData = stdenv.mkDerivation rec {
+    name = "tesseract-${version}";
+    version = "3.05.00";
 
-  enableParallelBuilding = true;
+    src = fetchFromGitHub {
+      owner = "tesseract-ocr";
+      repo = "tesseract";
+      rev = version;
+      sha256 = "11wrpcfl118wxsv2c3w2scznwb48c4547qml42s2bpdz079g8y30";
+    };
 
-  nativeBuildInputs = [ pkgconfig autoreconfHook ];
-  buildInputs = [ leptonica libpng libtiff icu pango opencl-headers ];
+    enableParallelBuilding = true;
 
-  LIBLEPT_HEADERSDIR = "${leptonica}/include";
+    nativeBuildInputs = [ pkgconfig autoreconfHook ];
+    buildInputs = [ leptonica libpng libtiff icu pango opencl-headers ];
 
-  postInstall = ''
-    for i in ${tessdata}/share/tessdata/*; do
-      ln -s $i $out/share/tessdata;
-    done
-  '';
+    LIBLEPT_HEADERSDIR = "${leptonica}/include";
 
-  meta = {
-    description = "OCR engine";
-    homepage = https://github.com/tesseract-ocr/tesseract;
-    license = stdenv.lib.licenses.asl20;
-    maintainers = with stdenv.lib.maintainers; [viric];
-    platforms = with stdenv.lib.platforms; linux ++ darwin;
+    passthru = { inherit languages; };
+
+    meta = {
+      description = "OCR engine";
+      homepage = https://github.com/tesseract-ocr/tesseract;
+      license = stdenv.lib.licenses.asl20;
+      maintainers = with stdenv.lib.maintainers; [viric];
```

Comment:
You might want to add yourself as a maintainer.

Reply:
[Done.](https://github.com/NixOS/nixpkgs/pull/52379/commits/412a5b33213a9bc07498b43beca81a5ec01dc5e4)